regression/scripts/lifting_line/lifting_line.py

```diff
@@ -160,9 +160,11 @@ def main():
     #compute_aircraft_lift(conditions, configuration, geometry) 
     
     lift = state.conditions.aerodynamics.lift_coefficient
```

Comment:
Would still like to see the explanation for these differences.

Reply:
In the past, the lifting line function had a conditional statement that was incorrect and therefore we did not catch that a wing with multiple segments must have a tip segment that has a percent_span_location of 1.0. When I corrected the vehicle geometry with a tip segment I discovered this error, this applying a small correction to the conditional statement. I did not change the functions themselves